library/src/helpers/parser.ts

```diff
@@ -17,13 +12,13 @@ type ParserOptions = AsyncApiProps['parserOptions'];
 type ParseDocument = (
   content: string | any,
   parserOptions?: ParserOptions,
-) => Promise<AsyncAPI>;
+) => Promise<any>;
```

Comment:
Because parsers function from `asyncapi-parser` return object `{ _json }` and also function `json()`, so I decide to serializing data to our interfaces and by that for future we will be immune to changes in api of `asyncapi-parser`.